.readme-partials.yaml

```diff
@@ -231,15 +231,15 @@ custom_content: |
   Cloud Logging libraries use [trace fields within LogEntry](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#FIELDS.trace) to capture trace contexts, which enables the [correlation of logs and traces](https://cloud.google.com/logging/docs/view/correlate-logs), and distributed tracing troubleshooting. 
   These tracing fields including [trace](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#FIELDS.trace), [spanId](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#FIELDS.span_id), and [traceSampled](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#FIELDS.trace_sampled) define the trace context for a `LogEntry`.
   
-  The library provides three methods to update the context:
+  The library provides three methods to update trace/span Id:
   * Manually set trace/span Id
   The trace and span Ids can be set directly using `setTraceId()` and `setSpanId()` methods provided by `Context.Builder`. Alternatively it can also be parsed from the [W3C tracing context header](https://www.w3.org/TR/trace-context) using `loadW3CTraceParentContext()` or from the [Google Cloud tracing context header](https://cloud.google.com/trace/docs/trace-context#legacy-http-header) using `loadCloudTraceContext()`.
```

Comment:
I'd change "set directly using" to "set directly by using"
I'd also change "Alternatively it can" to "Alertnatively, trace and span IDs can"  (I'm not sure what "it" refers to here, but I think you mean the trace/spanID values).
I'd also change "W3C tracing context header(link) using" to "W3C tracing context header(link) by using"
And "Google Cloud tracing context" should probably be "Cloud Trace context" (the official product name is "Cloud Trace")
and "using `loadCloudTraceContext()` to "by using `loadCloudTraceContext()`.